taskrunners/cleaner/src/Connect.py

```diff
@@ -0,0 +1,14 @@
+from pymongo import MongoClient
+
+def connect():
+    domain = 'domain.com'
+    username = 'user'
+    password = 'password'
+    authSource = 'auth'
+    return MongoClient(domain, 27017, username=username, password=password, authSource=authSource).finlab_beta
```

Comment:
Is there a reason this is finlab_beta? The other Connect.py has finlab_prod. Connect.py aren't consistent

Reply:
Updated to finlab_prod.